src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -27,15 +27,15 @@ function DisputeVoting({ isJurorDrafted, onRequestCommit }) {
           disabled={!isJurorDrafted}
           onClick={() => onRequestCommit(VOTE_OPTION_IN_FAVOR)}
         >
-          In favor
+          In favor of the plaintiff
         </VotingButton>
         <VotingButton
           mode="negative"
           wide
           disabled={!isJurorDrafted}
           onClick={() => onRequestCommit(VOTE_OPTION_AGAINST)}
         >
-          Against
+          Against the plaintiff
```

Comment:
I think this is from the previous PR? 😄

Reply:
haha yes seems that i created this one from the previous branch not develop hah